openlibrary/templates/admin/spamwords.html

```diff
@@ -4,34 +4,34 @@
 
 <div id="contentHead">
     $:render_template("admin/menu")
-    <h1>Spam Words</h1>
+    <h1>$_("Spam Words")</h1>
 </div>
 
 <div id="contentBody">
-    <h2>Spam Words</h2>
+    <h2>$_("Spam Words")</h2>
     <p>
-        Please enter the SPAM regular expressions in the textarea below, one per line.<br>
-        Be sure to escape any meta characters that are meant to be character literals.<br>
-        If you are adding a domain, prepend the following to the domain: <code>(\b|https://|http://)</code><br>
-        For example, if you want to prevent edits containing the domain <code>t.co</code>, add <code>(\b|https://|http://)t\.co</code> to the spamwords list.
+        $_("Please enter the SPAM regular expressions in the textarea below, one per line.")<br>
+        $_("Be sure to escape any meta characters that are meant to be character literals.")<br>
+        $_("If you are adding a domain, prepend the following to the domain:") <code>(\b|https://|http://)</code><br>
+        $:_('For example, if you want to prevent edits containing the domain <code>t.co</code>, add <code>(\b|https://|http://)t.co</code> to the spamwords list.')
```

Comment:
```suggestion
        $:_('For example, if you want to prevent edits containing the domain <code>t.co</code>, add <code>(\b|https://|http://)t\\.co</code> to the spamwords list.')
```